src/components/LayerKey.vue

```diff
@@ -83,9 +84,11 @@ export default {
     blur() {
       this.startListening();
       this.setSelected(undefined);
+      this.hasFocus = false;
     },
     focus() {
       this.stopListening();
+      this.hasFocus = true;
```

Comment:
ditto

Reply:
The same position absolute trick is applied here (because the scss is global and not scoped) so to avoid refactoring and renaming classes I decided to give it the same treatment but just not enable the input to expand as traditionally the layer input would be 0,1 or 2 chars wide which works already.